### PR TITLE
Minor changes to DifferentiationSNP.Rmd

### PR DESCRIPTION
**Pull request by zkamvar**
I made minor changes to @smanel's vignette for differentiation in SNPs for spelling and clarity.

### CODE REVIEW

**Comment by zkamvar** [comment]
Note: I'm planning on going through all the vignettes and ensuring that they have the same code style and no typos. @hlapp, should I create a separate pull request for each one?

**Comment by hlapp** [comment]
> @hlapp, should I create a separate pull request for each one?

I don't know. I have no objection to bundling up many small but related changes into one pull request. But if there's potential debate about some part that won't apply to many others then I'd argue it's better to keep things separated.

**Comment by zkamvar** [comment]
> if there's potential debate about some part that won't apply to many others then I'd argue it's better to keep things separated.

Good point. I'd better keep 'em separated.

**Comment by zkamvar** [comment]
That means this one is ready for review

File: use/DifferentiationSNP.Rmd [hunk 1]
```diff
@@ -62,23 +58,23 @@ dim(Mydata)
...
+dim(Mydata) # 550 individuals x 3082 SNPs
 ```
 
-Data conversion
+### Data conversion
 
-To convert MydataR to genind objects (adegent), the object locus needs to
-contain only genotypes. We decrease the number of snp to make the calculations
-faster and keep only 20 SNPs in the object locus. We convert Mydata1 in a fstat
+To convert Mydata to a genind object (adegenet), the input should only contain
```
Comment (hlapp) [inline use/DifferentiationSNP.Rmd]: I wonder whether we should consider and settle on consistency for formatting of R object names. Personally, I would put everything that's an R object, perhaps with the exception or R package names, in backtics, so they are clearly distinguished from English language words, and as a signal to the reader that these will occur or have occurred in, and thus directly relate to, the preceding or following code chunks. Right now, sometimes data structure references are not backtic'ed, sometimes they are, and function names mostly seem to be. Perhaps it's not too early to start making these consistent, especially for a PR like this that's about "linting".

**Comment by hlapp** [comment]
:+1: to all and see comment for consideration.

File: use/DifferentiationSNP.Rmd (hunk 1 above)
Comment (zkamvar) [inline use/DifferentiationSNP.Rmd]: This makes sense. Perhaps I'll create an additional pull request for a style sheet in CONTRIBUTING

File: use/DifferentiationSNP.Rmd [hunk 2]
```diff
@@ -52,33 +48,33 @@ AT...).
...
 
-Data conversion
+### Data conversion
 
-To convert MydataR to genind objects (adegent), the object locus needs to
-contain only genotypes. We decrease the number of snp to make the calculations
-faster and keep only 20 SNPs in the object locus. We convert Mydata1 in a fstat
-object (Mydata2)
+To convert `Mydata` to a "genind" object (*adegenet*), the input should only
+contain only genotypes. We decrease the number of SNPs to make the calculations
```
Comment (hlapp) [inline use/DifferentiationSNP.Rmd]: _only_ is repeated. (Sorry for the nitpicking :smile:)

File: use/DifferentiationSNP.Rmd [hunk 3]
```diff
@@ -120,22 +117,22 @@ genet.dist(Mydata1, method="WC84")
 ## Unsupervised clustering  
 
 We don't know the populations and we are looking for. As recommended by T.
-Jombart, with the function `gpr` we used the maximum  possible number of PCA
+Jombart, with the function `find.clusters()` we used the maximum  possible number of PCA
 axis which is 20 here. See detailed tutorial on this method for more information
 (http://adegenet.r-forge.r-project.org/) In this example, we used
```
Comment (hlapp) [inline use/DifferentiationSNP.Rmd]: Shouldn't the link be hyperlinked text instead? Also, the URL as given doesn't actually go to _tutorial on this method_, unlike what the text suggests. It goes to the adegenet website on R-Forge, which says that will soon move to Github (has it meanwhile?). Is the tutorial resource meant to be visited by the reader [this one on R-Forge](http://adegenet.r-forge.r-project.org/documentation.html) (the frame-based structure of the website on R-Forge is terrible for direct linking! cc @thibautjombart), or [this one on Github](https://github.com/thibautjombart/adegenet/wiki/Tutorials), or one of the ones listed on Github (or R-Forge).

Comment (zkamvar) [inline use/DifferentiationSNP.Rmd]: I'm going to make a direct link to the tutorial on GitHub. These are the ones that are up to date.

**Comment by hlapp** [comment]
Since the changes are solely minor in nature, stylistic and for consistency, I'm going to go ahead and call this good for merging.

**Pull request merged**